exercises/practice/matrix/.meta/example.sml

```diff
@@ -0,0 +1,16 @@
+local
+  val lines: string -> string list =
+    String.tokens (fn (c: char) => c = #"\n")
+
+  val cells: string -> string list =
+    String.tokens (fn (c: char) => c = #" ")
+
+    fun extract (index: int) (l: string list): string =
```

Comment:
Could you fix the indentation?

The `extract` function is more indented than the rest.

Reply:
done